app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -112,24 +112,47 @@
       = miq_tab_content('amqp', 'default') do
         .form-group
           .col-md-12{"ng-if" => "#{ng_model}" == "emsCommonModel" && "#{ng_model}.emstype == 'openstack' || #{ng_model}.emstype == 'openstack_infra'"}
-            = render :partial => "layouts/angular-bootstrap/endpoints_angular",
-                                 :locals  => {:ng_show                => true,
-                                              :ng_model               => "#{ng_model}",
-                                              :id                     => record.id,
-                                              :prefix                 => "amqp"}
-            = render :partial => "layouts/angular-bootstrap/auth_credentials_angular_bootstrap",
-                                 :locals  => {:ng_show           => true,
-                                              :ng_model          => "#{ng_model}",
-                                              :ng_reqd_password  => "#{ng_model}.amqp_userid != ''",
-                                              :ng_reqd_verify    => "#{ng_model}.amqp_userid != ''",
-                                              :validate_url      => validate_url,
-                                              :id                => record.id,
-                                              :prefix            => "amqp",
-                                              :basic_info_needed => true}
+            %label.radio-inline.control-label{"for" => "textInput-markup"}
+              %input{:type         => "radio",
+                     :name         => "event_stream_selection",
+                     :id           => "ceilometer_radio",
+                     "ng-checked"  => "emsCommonModel.event_stream_selection == ceilometer",
+                     :value        => "ceilometer",
+                     "ng-model"    => "emsCommonModel.event_stream_selection",
+                     "checkchange" => ""} Ceilometer
+            %label.radio-inline.control-label{"for" => "textInput-markup"}
```

Comment:
textInput-markup should be id of the input, so amqp_radio. same above, by knowledge of @himdel :-)